home_vending_machines/crafts.lua

```diff
@@ -19,7 +19,7 @@ if minetest.get_modpath("default") then
             },
         })
     end
-    if minetest.get_modpath("farming") then
+    if farming and farming.mod and farming.mod == "redo" then
```

Comment:
```suggestion
    if farming and farming.mod == "redo" then
```
Truth value check is unnecessary and also easier to read without it.